src/aleo.rs

```diff
@@ -0,0 +1,325 @@
+//! Functionality related to [Aleo] blockchain network.
+//!
+//! Required crate feature: `aleosig`
```

Comment:
That requires additional attributes and crate-wide config.

e.g. https://docs.rs/ssi/0.2.0/ssi/hash/fn.sha256.html does not show it.

Looks like the old way was to use `#[doc(cfg(…))]` in addition to `#[cfg(...)]`: https://doc.rust-lang.org/unstable-book/language-features/doc-cfg.html
as done in the elliptic-curves crates, gating the unstable `#[doc(cfg(…))]` with an attribute enabled for docs.rs:
https://github.com/RustCrypto/elliptic-curves/blob/750e84c0a3e00fc3342251f1e3c0db66fa169bc4/k256/src/ecdsa/recoverable.rs#L102-L104
https://github.com/RustCrypto/elliptic-curves/blob/750e84c0a3e00fc3342251f1e3c0db66fa169bc4/k256/Cargo.toml#L60

Since https://github.com/rust-lang/rust/pull/89596, it looks like `#[doc(cfg(…))]` is not needed explicitly, but by using features doc_cfg and doc_auto_cfg. Tokio is using this recently (Still using a docsrs attribute): https://github.com/tokio-rs/valuable/pull/80
That is working on docs.rs as docs.rs is using nightly:
https://docs.rs/valuable/latest/valuable/trait.Mappable.html#foreign-impls
https://docs.rs/crate/valuable/0.1.0/builds/486108

There's also a crate to abstract away the conditional on nightly: https://docs.rs/doc-cfg/0.1.0/doc_cfg/